components/tools/OmeroWeb/omeroweb/webgateway/views.py

```diff
@@ -2267,24 +2267,42 @@ def _annotations(request, objtype, objid, conn=None, **kwargs):
     query += """
```

Comment:
All the test I've done where on 5.1 so maybe there's a difference to how the objects are loaded now.